test/scripts/bridge/build-relayer.sh

```diff
@@ -15,7 +15,7 @@ if [ -d "$relayer_root_dir" ];
 then
   echo "Relayer seems to be already setup. Skipping git fetch"
 else
-  git clone https://github.com/Snowfork/snowbridge $relayer_root_dir
+  git clone https://github.com/moondance-labs/snowbridge $relayer_root_dir
```

Comment:
People who have already run this script need to remove the snowbridge folder, otherwise it will fail, as it will use the old remote. CI works?

Reply:
*disregard*